### PR TITLE
#903. Change focus on last element when auto-reloading search page

### PR DESCRIPTION
**Pull request by madryk**

### CODE REVIEW

File: saos-webapp/src/main/java/pl/edu/icm/saos/webapp/judgment/PageLinkGenerator.java [hunk 1]
```diff
@@ -23,19 +25,28 @@
 	 * @param request
 	 * @param pageNoParameterName A page number parameter name. 
 	 */
-	public static String generateSearchPageBaseLink(HttpServletRequest request, String pageNoParameterName) {
+	public static String generateSearchPageBaseLink(HttpServletRequest request, String pageNoParameterName, List<String> filterParameters) {
```
Comment (lukdumi) [inline saos-webapp/src/main/java/pl/edu/icm/saos/webapp/judgment/PageLinkGenerator.java]: filterParameters in a little misleading i think, they are not parameters that will filter something or that are filtered... hmm maybe excludeQueryParameters.

File: saos-webapp/src/main/webapp/WEB-INF/static/javascript/judgmentSearch.js [hunk 2]
```diff
@@ -3,8 +3,17 @@
  * 
  * @author Łukasz Pawełczak
  */
-var jsInitInJudgmentSearch = function() {
+var jsInitInJudgmentSearch = function(options) {
```
Comment (lukdumi) [inline saos-webapp/src/main/webapp/WEB-INF/static/javascript/judgmentSearch.js]: options parameter not described

File: saos-webapp/src/main/webapp/WEB-INF/static/javascript/modules/search/searchCriteria.js [hunk 3]
```diff
@@ -11,30 +11,85 @@ var SearchCriteria = {
...
 		init: function() {
-			SearchCriteria.assigPageSize();
+			SearchCriteria.saveInitialValues();
+			SearchCriteria.assignPageSize();
 			SearchCriteria.assignSort();
+			SearchCriteria.assignKeyPress();
+			SearchCriteria.assignTrackingFocus();
 		},
 		
+		saveInitialValues: function() {
```
Comment (lukdumi) [inline saos-webapp/src/main/webapp/WEB-INF/static/javascript/modules/search/searchCriteria.js]: A small comment would be nice...

Comment (madryk) [inline saos-webapp/src/main/webapp/WEB-INF/static/javascript/modules/search/searchCriteria.js]: ok

File: saos-webapp/src/main/webapp/WEB-INF/static/javascript/judgmentSearch.js (hunk 2 above)
Comment (madryk) [inline saos-webapp/src/main/webapp/WEB-INF/static/javascript/judgmentSearch.js]: will add some comment

File: saos-webapp/src/main/java/pl/edu/icm/saos/webapp/judgment/PageLinkGenerator.java (hunk 1 above)
Comment (madryk) [inline saos-webapp/src/main/java/pl/edu/icm/saos/webapp/judgment/PageLinkGenerator.java]: :+1: changed to parametersToExclude

**Pull request closed without merging**